zoo-css-challenge/style.css

```diff
@@ -30,7 +32,7 @@ section {
 }
 
 img {
-  border: 5px solid #8075ff;
+  border: none;
```

Comment:
```suggestion
```
We could also delete the whole `img` selector, as an `img` doesn't have a border by default

Reply:
Great, Thank you very much